src/AudioDevice.cpp

```diff
@@ -155,23 +156,31 @@ namespace earlyapp
         LINF_(TAG, "Start ALSA playback");
 
         // Prepare ALSA device.
-        if((pcm = snd_pcm_open(&pALSAHandle, "plughw:0,0", SND_PCM_STREAM_PLAYBACK, 0)) < 0)
-        {
+        while(1) {
+            static int cnt = 0;
```

Comment:
Is there a reason keeping this error counts with static? This variable will be kept if there are re-entering this function. Is this expected?

Reply:
As a local static variable inside while scope, without static, cnt will be set 0 in next loop. However, I should set to 0 before return.